reactor3/src/test/java/com/linecorp/armeria/common/reactor3/ContextAwareFluxTest.java

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.reactor3;
+
+import static com.linecorp.armeria.common.reactor3.ContextAwareMonoTest.ctxExists;
+import static com.linecorp.armeria.common.reactor3.ContextAwareMonoTest.newContext;
+import static com.linecorp.armeria.common.reactor3.ContextAwareMonoTest.noopSubscription;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.reactivestreams.Publisher;
+
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.common.util.SafeCloseable;
+import com.linecorp.armeria.internal.testing.AnticipatedException;
+
+import reactor.core.Disposable;
+import reactor.core.publisher.ConnectableFlux;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+import reactor.test.StepVerifier;
+
+class ContextAwareFluxTest {
```

Comment:
How about adding a test case for `ParallelFlux` and adding `@Ignore` to it?

Reply:
Can I add it when I implement the hook later?
It seems weird that adding test cases for what I didn't implement it yet. 😎

Reply:
No problem